Automattic/wp-calypso
# Remove Jetpack "Add new site" page/modal

First, make sure that changes from #45500 have taken effect without repercussions and that the `/jetpack/new` page has stopped receiving traffic.

#### Changes proposed in this Pull Request

* This PR deletes code associated with the Jetpack "New Site" page/modal, which has been already disabled in a previous PR (#45500)

#### Testing instructions

* From Calypso, create a new site (see #45500 for detailed instructions) — the first page shown should be the start of the dotcom site creation flow
* Visiting `/jetpack/new` should redirect the user to the start of the site creation flow
* Signup and login links that previously pointed at `jetpack/new` are now pointing to `/jetpack/connect`

Partially covers changes requested in #45486
